### PR TITLE
Fix end-to-end testing workflow

### PR DESCRIPTION
**Pull request by j-f1**
I looked at a few existing actions but they didn’t seem to work well for commit statuses (they’re designed for checks). I rolled my own since it didn’t seem too difficult.

### CODE REVIEW

**Comment by j-f1** [comment]
OK I think this is ready to go.

File: bin/netlify-wait-for-build [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+current_commit=$(git rev-parse HEAD)
+
+echo "Running on commit $COMMIT_SHA"
+
+function get_status() {
+  echo "::group::API Response"
+  curl --header "Authorization: Bearer $GITHUB_TOKEN" "https://api.github.com/repos/actualbudget/actual/commits/$COMMIT_SHA/statuses" > /tmp/status.json
```
Comment (MatissJanis) [inline bin/netlify-wait-for-build:9]: I think this won't work on PRs coming from forks (so basically every other contributor). They won't have access to our `GITHUB_TOKEN`

Comment (MatissJanis) [reply bin/netlify-wait-for-build:9]: Maybe I'm totally wrong tough. Mind testing it out from a fork?

Comment (j-f1) [reply bin/netlify-wait-for-build:9]: I can change it to `pull_request_target` which uses the workflow file from the `master` branch and therefore always has access to the token.

**Review by MatissJanis — APPROVED** [review]
Thanks for doing this!

**Pull request merged**